cmd/build.go

```diff
@@ -157,7 +171,53 @@ func newBuildCmd() *cobra.Command {
 				return err
 			}
 
-			api := apiNew(entry.APIURL, entry.Token)
+			if entry.UUID == "" {
+				fmt.Println("sd-local collects UUIDs for statistical surveys.")
+				fmt.Println("You can reset it later by removing the UUID key from config.")
+				fmt.Print("Would you please cooperate with the survey? [y/N]: ")
+
+				input, err := bufio.NewReader(os.Stdin).ReadString('\n')
+				if err != nil {
+					return err
+				}
+				input = strings.TrimSuffix(input, "\n")
+				if input == "y" || input == "Y" || input == "yes" || input == "Yes" {
+					uuidObj, err := uuid.NewUUID()
```

Comment:
Is it better to use [NewString](https://pkg.go.dev/github.com/google/uuid#NewString)?
[NewUUID](https://pkg.go.dev/github.com/google/uuid#NewUUID) said "In most cases, New should be used.".

Reply:
fix to use NewString